src/main/java/de/tum/in/www1/artemis/service/connectors/ContinuousIntegrationService.java

```diff
@@ -104,6 +104,8 @@ void createBuildPlanForExercise(ProgrammingExercise exercise, String planKey, Vc
      */
     void deleteBuildPlan(String projectKey, String buildPlanId);
 
+    // TODO: GitLab CI Server Notification Plugin
+
```

Comment:
Why is this ToDo in here? Having it only in the GitlabCiService should be enough.

```suggestion
```